### PR TITLE
Collections

### PR DESCRIPTION
**Pull request by Folkskrik**

### CODE REVIEW

File: csharp-basics/exercises/Collections/DecryptNumber/Program.cs [hunk 1]
```diff
@@ -16,6 +17,32 @@ static void Main(string[] args)
...
+
+            var decryptedNumbers = cryptedNumbers.Select(DecryptString);
+
+            foreach (var decryptedNumber in decryptedNumbers)
+            {
+                Console.WriteLine(decryptedNumber);
+            }
+        }
+
+        public static string DecryptString(string encryptedString)
```
Comment (ausmoons) [inline csharp-basics/exercises/Collections/DecryptNumber/Program.cs]: Šim nebūtu jābūt public, jo neizmanto citā failā. Private

File: csharp-basics/exercises/Collections/FlightPlanner/Program.cs [hunk 2]
```diff
@@ -9,15 +9,107 @@ namespace FlightPlanner
...
+                }
+
+                if (!flights.ContainsKey(cities.First()))
+                {
+                    flights[cities.First()] = new List<string>();
+                }
+                flights[cities.First()].Add(cities.Last());
+            }
+
+            while (true)
```
Comment (ausmoons) [inline csharp-basics/exercises/Collections/FlightPlanner/Program.cs]: while (true) -- mainīgo true vietā

File: csharp-basics/exercises/Collections/Histogram/Program.cs [hunk 3]
```diff
@@ -9,15 +9,32 @@ namespace Histogram
 {
     class Program
     {
-        private const string Path = "../Histogram/midtermscores.txt";
+        private const string Path = "C:\\Users\\aleks\\source\\repos\\CodeLex\\csharp-basics\\exercises\\Collections\\Histogram\\midtermscores.txt";
```
Comment (ausmoons) [inline csharp-basics/exercises/Collections/Histogram/Program.cs]: path vajag tādu, lai strādā arī uz citiem pc

File: csharp-basics/exercises/Collections/Histogram/Program.cs [hunk 4]
```diff
@@ -9,15 +9,32 @@ namespace Histogram
 {
     class Program
     {
-        private const string Path = "../Histogram/midtermscores.txt";
```
Comment (ausmoons) [inline csharp-basics/exercises/Collections/Histogram/Program.cs]: Šādāveidā path vajag

**Review by ausmoons — CHANGES_REQUESTED** [review]

File: csharp-basics/exercises/Collections/Histogram/Program.cs (hunk 3 above)
Comment (Folkskrik) [reply csharp-basics/exercises/Collections/Histogram/Program.cs]: Ja ir cits path tad man nestrādā.

File: csharp-basics/exercises/Collections/Histogram/Program.cs (hunk 4 above)
Comment (Folkskrik) [reply csharp-basics/exercises/Collections/Histogram/Program.cs]: Es saprotu bet ja es nelieku pilno path tad man nestrādā kods, met error.

Comment (ausmoons) [reply csharp-basics/exercises/Collections/Histogram/Program.cs]: Kādu tieši erroru?

Comment (Folkskrik) [reply csharp-basics/exercises/Collections/Histogram/Program.cs]: Unhandled Exception: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\aleks\source\repos\CodeLex\csharp-basics\exercises\Collections\Histogram\bin\Histogram\midtermscores.txt'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalReadAllText(String path, Encoding encoding, Boolean checkHost)
   at System.IO.File.ReadAllText(String path)
   at Histogram.Program.Main(String[] args) in C:\Users\aleks\source\repos\CodeLex\csharp-basics\exercises\Collections\Histogram\Program.cs:line 16

Comment (ausmoons) [reply csharp-basics/exercises/Collections/Histogram/Program.cs]: Jo tāda path nav C:\Users\aleks\source\repos\CodeLex\csharp-basics\exercises\Collections\Histogram\bin\Histogram\midtermscores.txt'. Jāsaliek parezi ../, lai meklētu īstajā mapē. Pieņemu, ka vēl vienu failu ārpus, jo tagad tu meklē iekš bin

Comment (Folkskrik) [reply csharp-basics/exercises/Collections/Histogram/Program.cs]: es nezinu kā taisīt pareizus saīsinātos Path

**Review by EdgarsSalmins — APPROVED** [review]

**Pull request merged**